plugin-server/src/utils/db/db.ts

```diff
@@ -1246,6 +1247,12 @@ export class DB {
         return events
     }
 
+    // PerformanceEvent
+
+    public async fetchPerformanceEvents(): Promise<RawPerformanceEvent[]> {
```

Comment:
Looks like this is just for testing? Can we put this in the test folder to make that clear? Otherwise if someone uses this irl they'll be pulling all events.

Reply:
Indeed we could (I think)